packages/report-server/src/reportBuilder/output/functions/default.ts

```diff
@@ -0,0 +1,5 @@
+import { Row } from '../../types';
```

Comment:
Don't wanna be too much of a pain, but always good to the have Copyright at the top of all files 👍 
```suggestion
/**
 * Tupaia
 * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
 */
 
import { Row } from '../../types';
```